lib/scroll-through-time.js

```diff
@@ -35,30 +43,42 @@ function getListener (direction) {
 export default {
 
   config: {
-    autoToggle: {
-      title: 'Auto Toggle',
-      description: 'Toggle on start.',
+    active: {
+      title: 'Scroll History Active',
+      description: 'Enable on Atom startup.',
       type: 'boolean',
       default: true
     },
     direction: {
       title: 'Scroll direction',
+      description: 'Set the scroll direction action. Must be up/down on devices without touchpad or sidescroll.',
       type: 'string',
-      default: 'left',
+      default: 'down',
       enum: [
         {value: 'left', description: 'Scroll left to undo, right to redo'},
         {value: 'right', description: 'Scroll right to undo, left to redo'},
         {value: 'up', description: 'Scroll up to undo, down to redo'},
         {value: 'down', description: 'Scroll down to undo, up to redo'},
       ],
+    },
+    keyboardState: {
+      title: 'Keyboard + Mouse Support',
```

Comment:
what does "mouse" has to do with this option? It's just about having a modifier key right?

Reply:
I guess I was just confused. From my initial testing of the plugin it didn't seem to work on my desktop at all. However now that I switch back to master branch it does work; I have no clue why it didn't work for me initially.